go.mod

```diff
@@ -12,4 +14,4 @@ require (
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
 )
 
-go 1.16
+require github.com/pmezard/go-difflib v1.0.0 // indirect
```

Comment:
Per release notes

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.